<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>841</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="findcomboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>21</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Agency FB&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>性别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>19</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Agency FB&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
